src/makeCalculator.js

```diff
@@ -4,7 +4,48 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(number) {
+      this.result += number;
+
+      return this;
+    },
+
+    subtract(number) {
+      this.result -= number;
+
+      return this;
+    },
+
+    multiply(number) {
+      this.result *= number;
+
+      return this;
+    },
+
+    divide(number) {
+      if (number !== 0) {
```

Comment:
It's good that you're checking for division by zero to avoid runtime errors. However, it might be useful to provide feedback when division by zero is attempted. For example, you could throw an error or log a message to indicate that the operation is not allowed.